test/unit/component/parser/xml/BpmnXmlParser.activiti.5_14_1.test.ts

```diff
@@ -52,6 +53,7 @@ describe('parse bpmn as xml for Activiti Designer 5.14.1', () => {
         },
       },
     });
+    /* eslint-enable @typescript-eslint/no-unsafe-assignment */
```

Comment:
**question:** adding a `as XXX` wouldn't help for type resolution? both for user guidance and eslint?
This remark applies to all test files were the @typescript-eslint/no-unsafe-assignment rule is disable in such context